development/dotnet-runtime-5.0/README

```diff
@@ -6,3 +6,5 @@ build device, cloud, and IoT applications.
 This package contains everything you need to run a .NET 5.0 application.
 If you want to build apps, install .NET 5.0 SDK (dotnet-sdk-5.0), which
 contains this runtime too.
+
+NOTE: This release has reached end of life, meaning it is no longer supported. It is recommended to move to a supported release, such as .NET 6.0. 
```

Comment:
This needs wrapping and a newline

Reply:
It's really a minor thing (not even an issue), so kindly please do the necessary changes and merge it, and I will use it for the next updates.
Thanks in advance.

Reply:
> It's really a minor thing (not even an issue)

Actually it is. We enforce this rule so that the page for the build on the web site renders properly.

I have updates these now and staged them on my branch, but please check your builds in future.